### PR TITLE
wpa_passphrase: add page

### PR DESCRIPTION
**Pull request by AlvaroGarciaJaen**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/wpa_passphrase.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# wpa_passphrase
+
+> Generate a WPA PSK from an ASCII passphrase for a SSID.
```
Comment (mebeim) [inline pages/linux/wpa_passphrase.md]:
```suggestion
> Generate a WPA-PSK key from an ASCII passphrase for a given SSID.
```

File: pages/linux/wpa_passphrase.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- Display WPA PSK for a SSID reading passphrase from stdin:
```
Comment (mebeim) [inline pages/linux/wpa_passphrase.md]:
```suggestion
- Compute and display the WPA-PSK key for a given SSID reading the passphrase from stdin:
```
I think only saying "display" could be interpreted as "the key is already stored somewhere". Let's say "*compute* and display" instead.

File: pages/linux/wpa_passphrase.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+`wpa_passphrase {{ssid}}`
```
Comment (mebeim) [inline pages/linux/wpa_passphrase.md]:
```suggestion
`wpa_passphrase {{SSID}}`
```

File: pages/linux/wpa_passphrase.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+- Display WPA PSK for a SSID with a specified passphrase:
```
Comment (mebeim) [inline pages/linux/wpa_passphrase.md]:
```suggestion
- Compute and display the WPA-PSK key for a given SSID with specifying the passphrase as an argument:
```
Same as the previous description basically.

File: pages/linux/wpa_passphrase.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+`wpa_passphrase {{ssid}} {{passphrase}}`
```
Comment (mebeim) [inline pages/linux/wpa_passphrase.md]:
```suggestion
`wpa_passphrase {{SSID}} {{passphrase}}`
```

**Comment by AlvaroGarciaJaen** [comment]
I'm not used to contribute, so let me know if I should do something different for the next time!

**Comment by mebeim** [comment]
@AlvaroGarciaJaen well, you can do just anything you feel like doing: 

- Add missing pages.
- Edit existing pages (add examples, reword, reorganize content).
- Add translations in another language for existing pages.
- Different or even more complicated stuff...

As the first contribution this looks pretty good to me! Nice job :slightly_smiling_face:

**Comment by AlvaroGarciaJaen** [comment]
@mebeim Sorry, I didn't mean to do that. Missclicked!

**Pull request merged**